index.html

```diff
@@ -290,7 +290,19 @@ <h2>About me</h2>
         </div>
       </div>
     </main>
-
+    <section class="contact-me" id="contact">
+      <h2>Contact me</h2>
+      <p>
+          If you have an application you are interested in developing, 
+          a feature that you need built or a project that needs coding. 
+          i'd love to help with it.</p>
+      <form class="text-me" id="#form" method="post" action="">
+          <input type="text" class="simple" placeholder="User name" maxlength="30" name="User name" required>
+          <input type="email" class="simple" placeholder="Email" id="e-mail" name="email" required>
+          <textarea class="text-area" placeholder="comment" name="comment" required></textarea>
```

Comment:
I think it is important to have a maximum number of characters a Textarea message should have. You can as well add a `maxlength="500"` to your <textarea> tag. 👍🏽